### PR TITLE
Draft#715

### LINKED ISSUE #715: Prevent 'Draft' label from getting truncated
**Is it a ?**

- [x] Bug Report

- [ ] Feature Request
    
- [ ] Chore
    
### Bug Report    

**Actual Behaviour**

The 'Draft' label is not being displayed correctly on some devices. It is getting truncated and making the UI look bad.

**Expected Behaviour**

It should be on one line, without truncation.

**Screenshot of the issue**

badui

**Android version and Phone Model**

Android Version: 6.0.1; Phone Model: Xiaomi Redmi 4A

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by mishuvs**
Fixes #715 

Changes: 
1. Added a save icon (hex code: `#555`) instead of the draft label.
2. Background is now set as light yellow (#FFF9C4) for events in state 'draft'.

Screenshots for the change: 

newdraft

### CODE REVIEW

**Comment by iamareebjamal** [comment]
The icon should be very small, grey and aligned to the bottom of the layout

File: app/src/main/res/layout/event_layout.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="?attr/selectableItemBackground"
+        android:background="@{ event.state.equals(`draft`) ? @color/event_state_draft : android.R.attr.selectableItemBackground }"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/event_layout.xml]: This will cause ripple effect to be disabled

Comment (mishuvs) [reply app/src/main/res/layout/event_layout.xml]: I have made changes. Please review.

**Comment by mishuvs** [comment]
@iamareebjamal Please review.

File: app/src/main/res/layout/event_layout.xml [hunk 2]
```diff
@@ -17,7 +17,7 @@
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="?attr/selectableItemBackground"
+        android:background="@{ event.state.equals(`draft`) ? @color/event_state_draft : 0 }"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/event_layout.xml]: This will again make the app lose ripple effect

Comment (mishuvs) [reply app/src/main/res/layout/event_layout.xml]: I have added the `selectableItemBackground` on line 39. As far as I remember, it was working when I checked. I will check once more.

Comment (iamareebjamal) [reply app/src/main/res/layout/event_layout.xml]: But `0` is a magic number, why is that used?

Comment (mishuvs) [reply app/src/main/res/layout/event_layout.xml]: It seemed to have no effect. And I read (saw) somewhere someone using this. Sorry for being silly, but I think data binding doesn't support `only if statement`. Or if any other solution you suggest.

Comment (iamareebjamal) [reply app/src/main/res/layout/event_layout.xml]: Use `selectableItemBackground` instead

**Comment by mishuvs** [comment]
@iamareebjamal Please review.

**Comment by iamareebjamal** [comment]
@mishuvs Please resolve conflicts

**Pull request merged**